src/thebelab.js

```diff
@@ -393,11 +393,44 @@ export function requestBinder({
     url = binderUrl + "/build/gh/" + repo + "/" + ref;
   }
   console.log("Binder build URL", url);
-  events.trigger("status", {
-    status: "building",
-    message: "Requesting build from binder",
-  });
-  return new Promise((resolve, reject) => {
+
+  return new Promise(async (resolve, reject) => {
+    // if binder already spawned our pod and we remember the creds, reuse it
+    let spawned_pod_url = window.localStorage.getItem(
+      "thebe_binder_pod_url_" + url
+    );
+    let spawned_pod_token = window.localStorage.getItem(
+      "thebe_binder_pod_token_" + url
+    );
+    if (
+      ignoreSavedPod === false &&
```

Comment:
ignoreSavedPod is undefined?

Reply:
Whoops, remnant of some other code that no longer exists. Removed.